packages/client-sdk-web/src/index.ts

```diff
@@ -95,6 +95,12 @@ import {
   TokenScope,
   ExpiresIn,
   ExpiresAt,
+  CacheName,
```

Comment:
do we also need to export `TokenScopes` here too?

Reply:
Good catch! fixed in https://github.com/momentohq/client-sdk-javascript/pull/598/commits/37fef37d3c8a3a47ee92595a9d3070de4bbf8d1c